docs/examples/expandedRowRender.tsx

```diff
@@ -87,14 +88,18 @@ const Demo = () => {
   if (expandedRowKeys.length) {
     const closeAll = () => setExpandedRowKeys([]);
     toggleButton = (
-      <button type="button" onClick={closeAll}>
+      <button type="button" onClick={ closeAll }>
```

Comment:
Don't update these lines.

Reply:
The code style in this file has been restored.